### PR TITLE
Fetch intercom token in wp-admin

### PR DESCRIPTION
**Pull request by rcdtroly**
### Why:
* Intercom token does not fetch automatically from Troly
### What:
* If login as admin, fetch token in `admin_init` action

### CODE REVIEW

File: includes/admin/controllers/class-wp99234-admin.php [hunk 1]
```diff
@@ -351,24 +351,24 @@ function on_save_post( $post_ID, $post, $update ){
...
-     */	
+     */
     function filter_row_actions( $actions, $post ){
 
         if( $post->post_type == WP99234()->_products->products_post_type ){
             $actions['wp99234_subs_product_id'] = sprintf(
-              'Troly ID: %s, Last Updated (UTC+0): %s', 
-              get_post_meta($post->ID, 'subs_id', true), 
+              'Troly ID: %s, Last Updated (UTC+0): %s',
+              get_post_meta($post->ID, 'subs_id', true),
```
Comment (atifempireone) [inline includes/admin/controllers/class-wp99234-admin.php]: why is the subs_id, ie the customer id being used here?

File: wp99234.php [hunk 2]
```diff
@@ -490,18 +490,27 @@ public function admin_scripts() {
...
+                    $user_subs_id = get_user_meta($current_user->ID, 'subs_id', true);
+                    if ($user_subs_id) {
+                        $endpoint = WP99234()->_users->get_update_endpoint_for_user_id($user_subs_id);
+                        $user_response = WP99234()->_api->_call($endpoint);
+
+                        if (isset($user_response->intercom_hmac)) {
+                            $userhash = $user_response->intercom_hmac;
+
+                            $website      = get_site_url();
+                            $hosted       = in_array($_SERVER['SERVER_ADDR'], array('103.18.108.127'), true) ? "true" : "false";
```
Comment (atifempireone) [inline wp99234.php]: why do we have an IP hardcode here? 

Should be set as a environment variable.

File: includes/admin/controllers/class-wp99234-admin.php (hunk 1 above)
Comment (rcdtroly) [reply includes/admin/controllers/class-wp99234-admin.php]: @atifempireone that was written before, so I use it as fallback also for earlier records
see:
Alt text

File: wp99234.php (hunk 2 above)
Comment (rcdtroly) [reply wp99234.php]: @atifempireone IP added to constant

**Pull request merged**